doc/example.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+var fs = require('fs');
+var path = require('path');
+var insertcss = require('insert-css');
+insertcss(fs.readFileSync(path.join(__dirname, '/../dist/date-time-group.css'), 'utf8'));
```

Comment:
Should this still be here, should it instead be a stylesheet include from `example.html` to allow removing the dependency entirely?